Include glibmm.h.

Fixes:

error: no member named 'build_filename' in namespace 'Glib'
error: no member named 'FILE_TEST_EXISTS' in namespace 'Glib'
error: no member named 'FILE_TEST_IS_DIR' in namespace 'Glib'
error: no member named 'file_test' in namespace 'Glib'

https://sourceforge.net/p/gelide/bugs/20/

https://abf.io/import/gelide/blob/rosa2021.1/gelide-0.1.5-glibmm.patch
--- src/config.hpp
+++ src/config.hpp
@@ -23,6 +23,7 @@
 #define _CONFIG_HPP_
 
 #include "gelide.hpp"
+#include <glibmm.h>
 #include <glibmm/ustring.h>
 #include <map>
 #include "utils/utils.hpp"
--- src/utils/utils.hpp
+++ src/utils/utils.hpp
@@ -24,6 +24,7 @@
 
 #include <sstream>
 #include <vector>
+#include <glibmm.h>
 #include <glibmm/ustring.h>
 #include "../gelide.hpp"
 #include <gdkmm/pixbuf.h>
